client/human/HumanClientFSM.h

```diff
@@ -291,15 +297,25 @@ struct WaitingForGameStart : boost::statechart::state<WaitingForGameStart, Playi
     typedef boost::statechart::state<WaitingForGameStart, PlayingGame> Base;
 
     typedef boost::mpl::list<
+        boost::statechart::custom_reaction<DoneLoading>,
         boost::statechart::custom_reaction<GameStart>
     > reactions;
 
     WaitingForGameStart(my_context ctx);
     ~WaitingForGameStart();
 
+    boost::statechart::result react(const DoneLoading& msg);
     boost::statechart::result react(const GameStart& msg);
 
     CLIENT_ACCESSOR
+
+    //! Load data in an asyc thread.
```

Comment:
what data? explain what this does / how / why.
should these functions be private?